tests/e2e/support/objects/app-admin-settings/spaces/actions.ts

```diff
@@ -8,6 +8,11 @@ const spaceCheckboxSelector = `[data-item-id="%s"]:not(.oc-table-highlighted) in
 const quotaActionBtn = `.oc-files-actions-edit-quota-trigger`
 const disableActionBtn = `.oc-files-actions-disable-trigger`
 const deleteActionBtn = `.oc-files-actions-delete-trigger`
+const enableActionBtn = `.oc-files-actions-restore-trigger`
+const renameActionBtn = `.oc-files-actions-rename-trigger`
+const editSubtitleActionBtn = `.oc-files-actions-edit-description-trigger`
```

Comment:
maybe use as:
`const actionBtn = '.oc-files-actions-%s-trigger'`

and after that use one button: 
`locator(util.format(actionBtn, 'edit-description')`

Reply:
@ScharfViktor I'll do it in this PR https://github.com/owncloud/web/pull/8629/files